deps/build.jl

```diff
@@ -24,11 +24,18 @@ if !libmxnet_detected
   ################################################################################
   # If not found, try to build automatically using BinDeps
   ################################################################################
-  if is_windows()
-    info("Please follow the libmxnet documentation on how to build manually")
-    info("or to install pre-build packages:")
-    info("http://mxnet.readthedocs.io/en/latest/how_to/build.html#building-on-windows")
-    error("Automatic building libxmnet on Windows is currently not supported yet.")
+  if is_windows()  
+	DOWNLOAD_URL = "https://github.com/dmlc/mxnet/releases/download/20160531/20160531_win10_x64_cpu.7z"
+	run(download_cmd(DOWNLOAD_URL, "mxnet.7z"))
+	run(`7z x mxnet.7z -y -ousr`)
+	run(`usr\\setupenv.cmd`)
+	run(`cmd /c copy "usr\\3rdparty\\openblas\\bin\\*.dll" "usr\\lib"`)
+	
+	DOWNLOAD_URL = "https://github.com/yajiedesign/mxnet/releases/download/20161125/20161125_mxnet_x64_cpu.7z"
```

Comment:
It is the function `MXSymbolGetAtomicSymbolName` that is available in the nightly version.

```
julia> using MXNet
INFO: Recompiling stale cache file C:\Users\julia\AppData\Local\JuliaPro 0 5 0 2
\pkgs-0.5.0.2\lib\v0.5\MXNet.ji for module MXNet.

WARNING: deprecated syntax "[a=>b for (a,b) in c]".
Use "Dict(a=>b for (a,b) in c)" instead.

WARNING: deprecated syntax "[a=>b for (a,b) in c]".
Use "Dict(a=>b for (a,b) in c)" instead.

WARNING: deprecated syntax "[a=>b for (a,b) in c]".
Use "Dict(a=>b for (a,b) in c)" instead.

WARNING: deprecated syntax "[a=>b for (a,b) in c]".
Use "Dict(a=>b for (a,b) in c)" instead.

WARNING: deprecated syntax "[a=>b for (a,b) in c]".
Use "Dict(a=>b for (a,b) in c)" instead.

WARNING: deprecated syntax "[a=>b for (a,b) in c]".
Use "Dict(a=>b for (a,b) in c)" instead.
ERROR: InitError: ccall: could not find function MXSymbolGetAtomicSymbolName in
library C:\Users\julia\AppData\Local\JuliaPro 0 5 0 2\pkgs-0.5.0.2\v0.5\MXNet\de
ps\usr\lib\libmxnet.dll
 in macro expansion at C:\Users\julia\AppData\Local\JuliaPro 0 5 0 2\pkgs-0.5.0.
2\v0.5\MXNet\src\base.jl:57 [inlined]
 in _get_atomic_symbol_name(::Ptr{Void}) at C:\Users\julia\AppData\Local\JuliaPr
o 0 5 0 2\pkgs-0.5.0.2\v0.5\MXNet\src\symbolic-node.jl:728
 in _populate_symbol_creator_cache!() at C:\Users\julia\AppData\Local\JuliaPro 0
 5 0 2\pkgs-0.5.0.2\v0.5\MXNet\src\symbolic-node.jl:737
 in __init__() at C:\Users\julia\AppData\Local\JuliaPro 0 5 0 2\pkgs-0.5.0.2\v0.
5\MXNet\src\base.jl:35
 in _include_from_serialized(::String) at .\loading.jl:150
 in _require_from_serialized(::Int64, ::Symbol, ::String, ::Bool) at .\loading.j
l:187
 in require(::Symbol) at .\loading.jl:394
during initialization of module mx
```